modules/installation-vsphere-regions-zones.adoc

```diff
@@ -0,0 +1,62 @@
+// Module included in the following assemblies:
+//
+//* installing-vsphere-installer-provisioned-customizations.adoc [IPI]
+//* installing-vsphere-installer-provisioned-network-customizations.adoc [IPI]
+//* installing-vsphere.adoc [UPI]
+//* installing-vsphere-network-customizations.adoc [UPI]
+//* installing-restricted-networks-installer-provisioned-vsphere.adoc [IPI]
+//* installing-restricted-networks-vsphere.adoc [IPI]
+
+:_content-type: CONCEPT
+[id="installation-vsphere-regions-zones_{context}"]
+= VMware vSphere region and zone enablement
+
+You can deploy an {product-title} cluster to multiple vSphere datacenters that run in a single VMware vCenter. Each datacenter can run multiple clusters. This configuration reduces the risk of a hardware failure or network outage causing your cluster to fail.
+
+[IMPORTANT]
+====
+The VMware vSphere region and zone enablement feature is only available with a newly installed cluster, because this feature requires the vSphere Container Storage Interface (CSI) driver as the default storage driver in the cluster.
+
+A cluster that was upgraded from a previous release defaults to using the in-tree vSphere driver, so you must enable CSI automatic migration for the cluster. You can then configure multiple regions and zones for the upgraded cluster. 
+====
+
+You must create an installation configuration file and then modify for the purposes of region and zone enablement, because deploying a cluster to a single vSphere datacenter is the default installation configuration.
```

Comment:
This is hard to follow due to the long sentence and some grammar issues. I recommend rewording as shorter sentences to clear things up.
```suggestion
The default installation configuration deploys a cluster to single vSphere datacenter. If you want to deploy a cluster to multiple vSphere data centers, you must create an installation configuration file that enables the region and zone feature.
```
I think that this changes the meaning, but I hope that it illustrates my point. :)